include/fmi4c.h

```diff
@@ -23,11 +23,12 @@
 #undef fmiVersion
 
 #ifdef _WIN32
-#ifdef FMI4C_DLLEXPORT
+#if defined(FMI4C_STATIC)
+#define FMI4C_DLLAPI
+#elif defined(FMI4C_DLLEXPORT)
 #define FMI4C_DLLAPI __declspec(dllexport)
 #else
-// TODO Maybe use dllimport
-#define FMI4C_DLLAPI
+#define FMI4C_DLLAPI __declspec(dllimport)
```

Comment:
I am not sure if dllimport is actually needed. It might be possible to leave this macro blank for both the static and dllimport case. Then the consumer would not need to define FMI4C_STATIC when a static library is used.